### PR TITLE
[CONTRIBUTING] add trivial change clause

### PR DESCRIPTION
**Pull request by macfarla**
## PR description
Adds a clause covering trivial changes. We've seen an increase in this type of PR lately (typos, punctuation, and other trivial changes). These add load to CI and also to reviewers. Until now we did not have a policy on these types of PRs. 

These words came from [Teku](https://github.com/Consensys/teku/blob/master/CONTRIBUTING.md)

> Please keep in mind that we do not accept non-code contributions like fixing comments, typos or some other trivial fixes. Although we appreciate the extra help, managing lots of these small contributions is unfeasible, and puts extra pressure in our continuous delivery systems (running all tests, etc). Feel free to open an issue pointing any of those errors, and we will batch them into a single change.

### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Locally, you can run these tests to catch failures early:

- [ ] unit tests: `./gradlew build`
- [ ] acceptance tests: `./gradlew acceptanceTest`
- [ ] integration tests: `./gradlew integrationTest`
- [ ] reference tests: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Having the following accounts is necessary for contributing code/issues to Besu.
 * To ask questions or chat with us, join our [Discord](https://discord.com/invite/hyperledger)
 * To edit pages in our wiki, you'll need a [Linux Foundation (LF) account].
 
+Please keep in mind that we do not accept non-code contributions like fixing comments, typos or some other trivial fixes. Although we appreciate the extra help, managing lots of these small contributions is unfeasible, and puts extra pressure in our continuous delivery systems (running all tests, etc). Feel free to open an issue pointing any of those errors, and we will batch them into a single change.
```
Comment (siladu) [inline CONTRIBUTING.md:14]: If you add a header section, you should be able to link directly to it.

**Review by siladu — APPROVED** [review]
LGTM, linkable header section would be a minor improvement

**Review by lu-pinto — APPROVED** [review]
Even though there's some helpful comment fixes it can be a bit of a gray area what to and what not to allow, so I think not allowing them altogether is what makes most sense from the POV of time/resources wasted with little benefit.
The approach to just create issues for typos is a good one IMO.

**Review by daniellehrner — APPROVED** [review]

**Review by fab-10 — APPROVED** [review]

**Review by garyschulte — APPROVED** [review]
🚢

**Review by jframe — APPROVED** [review]

**Review by jflo — APPROVED** [review]

**Pull request merged**